### PR TITLE
Multibyte character support for Exporter::shortenedExport()

### PR DESCRIPTION
**Pull request by iteman**
Currently, long multibyte string are garbled with `Exporter::shortenedExport()` as the following:

garbled-string

This PR provides multibyte character support for `Exporter::shortenedExport()`.

BTW, the test data is [the Iroha](http://en.wikipedia.org/wiki/Iroha) as [hiragana](http://en.wikipedia.org/wiki/Hiragana).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
         "php": ">=5.3.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0"
+        "phpunit/phpunit": "~4.0",
+        "ext-mbstring": "*"
```
Comment (whatthejeff) [inline composer.json]: Might be better to use `@requires extension mbstring` for that one test.

Comment (GrahamCampbell) [inline composer.json]: :+1:

**Comment by sebastianbergmann** [comment]
Somehow we / I forgot to merge this and now it conflicts with `master`. Can you rebase, @iteman?

**Comment by iteman** [comment]
@sebastianbergmann This branch has been rebased now.

**Comment by sebastianbergmann** [comment]
Merged manually, thanks.

**Comment by iteman** [comment]
@sebastianbergmann @whatthejeff Thank you :smile:

**Pull request closed without merging**